tests/vppoutputencode.cpp

```diff
@@ -184,6 +193,7 @@ bool VppOutputEncode::output(const SharedPtr<VideoFrame>& frame)
     }
     do {
         status = m_encoder->getOutput(&m_outputBuffer, drain);
+        getFrameDelay(m_outputBuffer);
```

Comment:
Need change to 
if (status == ENCODE_SUCCESS)
getFrameDelay(m_outputBuffer);
That's the reason why you may get same timeStamp multiple time.